fastpass/src/main/scala/scala/meta/internal/fastpass/pantsbuild/BloopPants.scala

```diff
@@ -605,6 +606,12 @@ private class BloopPants(
             ) ++ extraJvmOptions
           ),
           target.mainClass,
+          runtimeJavaHome.map(_ =>
+            C.JvmConfig(
+              runtimeJavaHome,
+              List(s"-Duser.dir=$workspace") ++ extraJvmOptions
```

Comment:
Can we reuse this logic with the compile-time JVM config?

Reply:
Done!